index.html

```diff
@@ -0,0 +1,208 @@
+                </div>
+
+                <div class="place">
+                    @ In some place in México to be defined soon
+                </div>
+            </div>
+        </div>
+        
+    
+        <div class="main-program">
+            <h2 class="subtitle">Main program</h2>
+            <div class="line"></div>
+            <div class="wrapper-items">
+                <div class="program-item">
+                    <img src="./images/ponents_icon.png" class="icon-main-program">
+                    <div class="activity">Lecture</div>
+                    <div class="activity-info">Listen to speakers from various countries about the recent advancements in electromagnetic theory.</div>
+                </div>
+                <div class="program-item">
+                    <img src="./images/exhibition_icon.png" class="icon-main-program">
+                    <div class="activity">Exhibition</div>
+                    <div class="activity-info">Appreciate new and old theories applying the concept of Faraday's tube by renowed scientists
+                    </div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/forum_icon.png" class="icon-main-program">
+                    <div class="activity">Forum</div>
+                    <div class="activity-info">Have the time to share your thoughts and opinions with experts for each topic.</div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/workshop_icon.png" class="icon-main-program">
+                    <div class="activity">Workshop</div>
+                    <div class="activity-info">Try creating your own work using your own eperiments rather than just watching at it.</div>
+                </div>
+                
+                <div class="program-item">
+                    <img src="./images/ignite.png" class="icon-main-program">
+                    <div class="activity">Ignite</div>
+                    <div class="activity-info">Get opportunities to network with CC affiliates around the world, also after the summit.</div>
+                </div>                
+                
+            </div>
+            
+            
+            <button class="orange-button">
+                Join CC Global Summit 2015
+            </button>
+
+            <div> <a href="#"> See the whole program </a></div>
+            
+    
+        </div>
+            
+        
+        <div class="title-speakers">
+            <h2 class="text-style"> Featured Speakers</h2>
+            <div class="line"></div>
+        </div>
+                    
+
+        <div class="speakers"> </div>
+        <div class="speakers-bis"> </div>
+        <div class="more">More <i id="more-icon" class="bi bi-chevron-down" style="color: red; "></i> </div>        
+        <div class="less">Less <i id="less-icon" class="bi bi-chevron-up" style="color: red; "></i> </div>
```

Comment:
- Kindly your speaker list has unnecessary gab compare to the given design please fix it 👌👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/210960085-eb4854b6-16dc-4067-8ee7-e1e0a79c2b5c.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/210960098-1fbac7d6-15f5-4f77-992d-33f0fda7ed8b.png" width="200">|


😎️

🤝️